src/buildifier/buildifier.ts

```diff
@@ -191,6 +203,10 @@ export function executeBuildifier(
   return new Promise((resolve, reject) => {
     // Determine the executable
     let executable = getDefaultBuildifierExecutablePath();
+    const buildifierConfigJsonPath = getDefaultBuildifierJsonConfigPath();
+    if (buildifierConfigJsonPath.length !== 0) {
```

Comment:
afaict, `bazelConfig.get<string>` and thereby `getDefaultBuildifierJsonConfigPath` can return `undefined`. Accessing `undefined.length` would lead to an error. We should probably use

```
  return bazelConfig.get<string>("buildifierConfigJsonPath") ?? "";
```

or something similar

Reply:
Looking at the [documentation](https://code.visualstudio.com/api/references/vscode-api#WorkspaceConfiguration), it does seem that `get<T>()` can return `undefined`. But why is this not an issue for the (existing) `getDefaultBuildifierExecutablePath()`? In any case, I'll go ahead and change this to use the overload with a default to fallback to `""` instead of `undefined`.